### PR TITLE
Change namespace to default

### PR DESCRIPTION
**Pull request by lisyukanna**

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -118,7 +118,7 @@ spec:
     Supported operating systems:
       - RHEL 7.x (x86 architecture)
 
-    > **Note**: The operator must be installed in the `kube-system` project.
+    > **Note**: The operator must be installed in the `default` project.
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml:121]: forgot to mention that we are not allowed to change old CSV yamls

File: vendor/github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1/openapi_generated.go [hunk 2]
```diff
@@ -11646,7 +11646,7 @@ func schema_k8sio_api_core_v1_ServiceProxyOptions(ref common.ReferenceCallback)
 					},
 					"path": {
 						SchemaProps: spec.SchemaProps{
-							Description: "Path is the part of URLs that include service endpoints, suffixes, and parameters to use for the current proxy request to service. For example, the whole request URL is http://localhost/api/v1/namespaces/kube-system/services/elasticsearch-logging/_search?q=user:kimchy. Path is _search?q=user:kimchy.",
+							Description: "Path is the part of URLs that include service endpoints, suffixes, and parameters to use for the current proxy request to service. For example, the whole request URL is http://localhost/api/v1/namespaces/default/services/elasticsearch-logging/_search?q=user:kimchy. Path is _search?q=user:kimchy.",
```
Comment (oriyarde) [inline vendor/github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1/openapi_generated.go:11649]: forgot to mention that the vendor dir is not relevant for this PR

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml (hunk 1 above)
Comment (lisyukanna) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml]: done

File: vendor/github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1/openapi_generated.go (hunk 2 above)
Comment (lisyukanna) [reply vendor/github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1/openapi_generated.go]: done

**Pull request merged**